app/actions/ControlActions.js

```diff
@@ -451,17 +454,34 @@ export const REVOKETICKETS_ATTEMPT = "REVOKETICKETS_ATTEMPT";
 export const REVOKETICKETS_FAILED = "REVOKETICKETS_FAILED";
 export const REVOKETICKETS_SUCCESS = "REVOKETICKETS_SUCCESS";
 
-export const revokeTicketsAttempt = (passphrase) => (dispatch, getState) => {
+export const revokeTicketsAttempt = (passphrase) => async (
+  dispatch,
+  getState
+) => {
   dispatch({ type: REVOKETICKETS_ATTEMPT });
-  return wallet
-    .revokeTickets(sel.walletService(getState()), passphrase)
-    .then((revokeTicketsResponse) => {
-      setTimeout(() => {
-        dispatch(getStakeInfoAttempt());
-      }, 4000);
-      dispatch({ revokeTicketsResponse, type: REVOKETICKETS_SUCCESS });
-    })
-    .catch((error) => dispatch({ error, type: REVOKETICKETS_FAILED }));
+  const walletService = sel.walletService(getState());
+  const accounts = sel.balances(getState());
+  const accountUnlocks = [];
+  accounts.map((acct) => {
```

Comment:
Switch from `map` to `forEach` since you're not using the output of the map (this avoids creating an unnecessary result object that ends up discarded)